src/pca.js

```diff
@@ -110,6 +110,22 @@ export class PCA {
     return predictions.subMatrix(0, predictions.rows - 1, 0, nComponents - 1);
   }
 
+  /**
+   * Calculates the inverse PCA transform
+   * @param {Matrix} dataset
+   * @param {Array} mean
+   * @return {Matrix} dataset projected in the PCA space
+   */
+  invert(dataset, mean) {
+    dataset = new Matrix(dataset);
+
+    var meanPlaceholder = new Array(dataset.length).fill(0);
```

Comment:
`dataset.rows`

In `Matrix` 6.0.0, the `Matrix` class no longer extends Array. It was causing too much trouble with transpilation.